charmcraft.yaml

```diff
@@ -7,6 +7,7 @@ platforms:
   ubuntu@20.04:arm64:
   ubuntu@22.04:amd64:
   ubuntu@22.04:arm64:
+  ubuntu@22.04:s390x:
```

Comment:
@carlcsaposs-canonical , ccc cache is still missing for s390x: https://github.com/canonical/mysql-test-app/actions/runs/16566628293/job/46848227470#step:11:26